tox.ini

```diff
@@ -69,7 +69,7 @@ commands =
 [testenv:unit]
 description = Run unit tests
 commands_pre =
-    poetry install --only main,charm-libs,unit --no-root
+    poetry install --only charm-libs,unit --no-root
```

Comment:
Had issues installing psycopg3 locally for the unit tests, may end up not being necessary.